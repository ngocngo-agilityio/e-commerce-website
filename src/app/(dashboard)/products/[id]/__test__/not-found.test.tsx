import { render } from '@testing-library/react';

// Components
import NotFound from '../not-found';

describe('NotFound', () => {
  it('should match snapshot', () => {
    const { container } = render(<NotFound />);

    expect(container).toMatchSnapshot();
  });
});
