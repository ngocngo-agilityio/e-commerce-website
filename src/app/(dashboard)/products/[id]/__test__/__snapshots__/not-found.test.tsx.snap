// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `
<div>
  <div
    class="css-qxo1c5"
  >
    <h2
      class="chakra-heading css-0"
    >
      404 Not Found
    </h2>
    <p
      class="chakra-text css-0"
    >
      Could not find the requested product.
    </p>
    <a
      href="/"
      style="text-decoration: underline;"
    >
      Return Home
    </a>
  </div>
</div>
`;
